src/darwin/Framework/CHIPTests/TestHelpers/MTRTestCase+ServerAppRunner.m

```diff
@@ -61,7 +61,7 @@ + (NSTask *)doStartAppWithName:(NSString *)name arguments:(NSArray<NSString *> *
         @"--passcode",
         [NSString stringWithFormat:@"%llu", passcode.unsignedLongLongValue],
         @"--KVS",
-        [NSString stringWithFormat:@"/tmp/chip-%@-kvs%u", name, uniqueIndex],
+        [NSString stringWithFormat:@"/tmp/xctest-%d-chip-%@-kvs%u", getpid(), name, uniqueIndex],
```

Comment:
We should consider making a similar change for the outFile/errorFile bits below.  Followup is fine for that.

Reply:
We could. For logs it's not really a problem to overwrite the existing file, but for the KVS we don't overwrite.